ivre/db/mongo.py

```diff
@@ -721,6 +721,55 @@ def getscreenshot(port):
         if url == "field":
             return port.get('screendata')
 
+    def set_label(self, flt, group, label, archives=False):
```

Comment:
Sorry I missed that, please use `archive` and not `archives` (I know that's already inconsistent in the current code, but that should be fixed in another PR).